src/Symfony/Component/PropertyAccess/Mapping/Cache/CacheInterface.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyAccess\Mapping\Cache;
+
+use Symfony\Component\PropertyAccess\Mapping\ClassMetadata;
+
+/**
+ * Persists ClassMetadata instances in a cache.
+ *
+ * @author Luis Ramón López <lrlopez@gmail.com>
+ */
+interface CacheInterface
```

Comment:
Metadata caching is still WIP. I just migrated the Validator component cache into PropertyAccess. As you say, the next logical step is integrate with the new Cache component